### ISSUE TITLE
v1.9.4 is not pushed to docker hub

### ISSUE THREAD

**Issue by w1ndy** [democratic-csi/democratic-csi#425]
The latest version on docker hub is still v1.9.3
https://hub.docker.com/r/democraticcsi/democratic-csi/tags

**Comment by disconn3ct** [comment]
Not pushed anywhere. https://github.com/democratic-csi/democratic-csi/pkgs/container/democratic-csi is also 1.9.3.

**Comment by travisghansen** [comment]
I had an issue with ci, as I recall it was only a doc update though so I didn’t bother getting images created. I could be wrong though..

**Comment by disconn3ct** [comment]
https://github.com/democratic-csi/democratic-csi/compare/v1.9.3...v1.9.4 agrees; it is trivial text/docs changes, nothing functional.

**Comment by w1ndy** [comment]
I believe the commit 7595191 fixes an important typo that prevents exceptions from throwing when a nvme subsystem is already connected...

**Comment by masterkain** [comment]
is this project maintained?

**Comment by travisghansen** [comment]
Yes. Most of the drivers are feature complete at this point but I have plans to introduce more in the coming months (mostly distributed node local stuff).

**Comment by masterkain** [comment]
allright thanks, I'm just getting my feet wet with truenas, do you suggest going with ssh drivers right now or rest apis ones (given they will be removed)?

**Comment by travisghansen** [comment]
I would use ssh. If you have any official docs referring to the removal of the REST api from the TN project I would love to have them added to this issue (or a new one). I have yet to see any official word on that but admittedly haven't gone hunting for it either.

**Comment by travisghansen** [comment]
For the others on this thread, I will update my CI environment to TN 24.10 and address whatever issues it may have and ensure a new release is snapped which includes the fix from 1.9.4.

**Comment by masterkain** [comment]
for now it seems they are deprecating the rest api with the next release, but removal date is not announced, https://www.truenas.com/docs/scale/api/

**Comment by travisghansen** [comment]
OK thanks for the links. I won't use the client but have plenty of familiarity with websockets so it's unlikely to be a huge hurdle.
